POC: Increase utilities test coverage (#224)
BBC-archive/psammead: packages/utilities/gel-foundations/README.md

## Contributing

> When **adding** a new export to this utility package the [export tests](https://github.com/BBC-News/psammead/blob/5d7395fd60bd8d73796d5a23775b4b5b36db1445/packages/utilities/gel-foundations/index.test.jsx#L13-L59) also need to be updated. When **removing** an exisiting export from this utility package the [export tests](https://github.com/BBC-News/psammead/blob/5d7395fd60bd8d73796d5a23775b4b5b36db1445/packages/utilities/gel-foundations/index.test.jsx#L13-L59) need to be updated and the package version requires a major change (EG: 1.2.1 -> 2.0.0) as this would be considered a breaking change as functionality is being removed.

Reviewer: ```suggestion
When **adding** a new export to this utility package, the export must be added to the [export tests](https://github.com/BBC-News/psammead/blob/5d7395fd60bd8d73796d5a23775b4b5b36db1445/packages/utilities/gel-foundations/index.test.jsx#L13-L59). When **removing** an existing export from this utility package, the export will need removing from the export tests and the package version requires a major change (EG: 1.2.1 -> 2.0.0).
```

(no need for repetition of the link)
